### ISSUE TITLE
Docker container tags to match nginx version.

### ISSUE THREAD

**Issue by wildone** [opentracing-contrib/nginx-opentracing#178]
Is it possible to have tags for the docker container to also have Nginx versions so it's easier to keep track of what version is running?

**Comment by miry** [comment]
@wildone Good idea. If you can play with https://github.com/opentracing-contrib/nginx-opentracing/blob/cb2f415f39bf969db24b0093b183cb6b4e4e24ed/ci/do_ci.sh#L32 and have a proposal of tags.

**Comment by wildone** [comment]
You are setting tags manually? If so need to get that process to set the tag to match the version of docker image nginx:latest at the time the pipeline is executed.

CI pickup tag from git `VERSION_TAG="`git describe --abbrev=0 --tags`"`
